ephemeris_tools_unit_tests/ephem_tools_unit_tests.py

```diff
@@ -0,0 +1,702 @@
+                    
+        # This check for HTML within the URLs is necessary to ensure that all tests
+        # Will function as expected. If tests are derived from a format other than
+        # HTML, this program will end and will return a logged warning.
+        logging.info('Compatibility check successful')
+            
+        ps_files = []
+        all_urls = list(viewer_urls)
+        number_of_base_tests = str(len(all_urls))
+        logging.info(f'Golden copies of the Planet Viewer Tool requested. Now '
+                     f'generating {number_of_base_tests} HTML file versions')
+    
+        logging.info(f'HTML test versions generated. Now generating '
+                     f'{number_of_base_tests} PostScript file versions')
+        
+        for url in viewer_urls: 
+            url = url.replace('output=HTML', 'output=PS')
+            ps_files.append(url)
+            all_urls.append(url)
+    
+        for file in all_urls:  
+            if file.endswith('output=HTML'):
+                index = list(viewer_urls).index(file) + 1
+            else:
+                assert file.endswith('output=PS')
+                index = ps_files.index(file) + 1
+            name = uuid.uuid5(uuid.NAMESPACE_URL, file)
+            file = 'https://staging.pds.seti.org/' + file
+            if 'test' in ephem:
+                file += '&ephem=+-1+TEST&sc_trajectory=+-1+TEST'
+            grab = requests.get(file, verify=False)
+            content = grab.content
+            content = content.decode('utf8')
+            with open(golden_copies_path[0] + '/' + str(name),
+                      'w') as golden_file:
+                golden_file.write(content)
+                logging.info(f'File {name} at line {index} generated')
+            
+    for tool in tools:
+        tests_file = os.path.join(source_location[0], tool)
+        with open(tests_file, 'r') as test_urls:
+            test_urls = test_urls.readlines()
+    
+        if 'viewer3' in test_urls[0]:
+            viewer_replace(ephem, tests_file, golden_copies_location)
+        if 'tracker3' in test_urls[0]:
+            moon_tracker_replace(ephem, tests_file, golden_copies_location)
+        if 'ephem3' in test_urls[0]:
+            ephemeris_generator_replace(ephem, tests_file, 
+                                        golden_copies_location)
+            
+current_time = datetime.now().strftime('%H:%M:%S')
+logging.basicConfig(filename=f'ephem_tools_unit_test_{current_time}.log',
+                    encoding='utf-8',
+                    level=logging.DEBUG, 
+                    format='%(asctime)s | %(levelname)s | %(message)s',
+                    datefmt='%y-%m-%d %H:%M:%S',
+                    force=True)
+    
+warnings.simplefilter('ignore', urllib3.exceptions.InsecureRequestWarning)
+logging.getLogger('urllib3').setLevel(logging.WARNING)
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--run-test-ephemeris', dest='chosenephems', 
```

Comment:
Using `append_const` seems like an odd thing to do here. All you want is a boolean flag saying whether you're using the test or the current ephemeris, right? So just make it a boolean called `use_test_ephemeris`. Set the default to `True` and make `--run-test-ephemeris` store `True`, which won't actually change anything, and make `--run-current-ephemeris` store `False`.